tools/mo/unit_tests/mo/utils/convert_impl_tmp_irs_cleanup_test.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import os
+import unittest
+from unittest.mock import patch
+
+from openvino.tools.mo.convert import convert_model
+from openvino.tools.mo.utils.error import Error
+
+
+def base_args_config():
+    args = argparse.Namespace()
+    args.extensions = [os.getcwd()]
+    args.use_legacy_frontend = False
+    args.use_new_frontend = False
+    args.framework = 'tf'
+    args.model_name = None
+    args.input_model = None
+    args.silent = True
+    args.transform = []
+    args.scale = None
+    args.output = None
+    args.input = None
+    args.input_shape = None
+    args.batch = None
+    args.mean_values = ()
+    args.scale_values = ()
+    args.output_dir = os.getcwd()
+    args.freeze_placeholder_with_value = None
+    args.transformations_config = None
+    args.disable_fusing = None
+    args.finegrain_fusing = None
+    args.disable_resnet_optimization = None
+    args.enable_concat_optimization = None
+    args.static_shape = None
+    args.disable_weights_compression = None
+    args.reverse_input_channels = None
+    args.data_type = None
+    args.layout = ()
+    args.source_layout = ()
+    args.target_layout = ()
+    args.input_checkpoint = None
+    args.saved_model_dir = None
+    args.input_meta_graph = None
+    args.saved_model_tags = None
+    args.progress = True
+    args.stream_output = False
+    args.tensorflow_use_custom_operations_config = None
+    args.tensorflow_custom_layer_libraries = None
+    args.tensorflow_custom_operations_config_update = None
+    args.tensorboard_logdir = None
+    args.disable_nhwc_to_nchw = False
+    return args
```

Comment:
this stuff looks copy-pasted. Can you import it?

Reply:
firstly i tried to import, but this is lead to importing of unnecessary modules from other places. In all tests arguments are redefined instead of importing: prior to my changes already 6 redefinitions of `base_args_config` and 0 imports of it.
![image](https://user-images.githubusercontent.com/5703039/208471121-162ac5c3-d0d4-4947-b121-77dfe521c607.png)
I propose to keep it as is

Reply:
please create utils.py in tests for such routines

Reply:
done